docker/Dockerfile

```diff
@@ -1,6 +1,11 @@
 ARG NODE_VERSION
 ARG DOCKERHUB=dockerhub.tax.service.gov.uk
-FROM ${DOCKERHUB}/node:${NODE_VERSION:-10.15.1}-slim
+FROM ${DOCKERHUB}/node:${NODE_VERSION}-bullseye-slim
+
+RUN sed -i 's/http:/https:/g' /etc/apt/sources.list
+
+RUN echo 'Acquire::https::Verify-Peer "false";' >/etc/apt/apt.conf.d/80-ignore-tls
+RUN apt-get update && apt-get install -y ca-certificates && rm -f /etc/apt/apt.conf.d/80-ignore-tls
```

Comment:
why is this little dance needed?

Reply:
I _think_ it's because traffic on Jenkins is going through a MITM style proxy

Reply:
I got this wrong 🤦‍♂️ we can actually reduce this whole PR down to just removing the apt-get update that's being done all together

Reply:
to add extra context to why, apt-get update is just fetching the latest list of stuff that could be installed, it doesn't actually update anything itself, so unless you are going to use apt to install something after there's no need for it - we only do an npm install after so there's no need for it and that was the only thing that triggered any http requests